src/el/besu/besu_launcher.star

```diff
@@ -231,15 +231,13 @@ def get_config(
         "--min-gas-price=1000000000",
         "--bonsai-limit-trie-logs-enabled=false",
     ]
-    if (
-        network not in constants.PUBLIC_NETWORKS
-        or constants.NETWORK_NAME.shadowfork in network
-    ):
+    if network not in constants.PUBLIC_NETWORKS:
         cmd.append(
             "--genesis-file="
             + constants.GENESIS_CONFIG_MOUNT_PATH_ON_CONTAINER
             + "/besu.json"
         )
+        cmd.append("--bonsai-limit-trie-logs-enabled=false")
```

Comment:
@parithosh just yeet this. This should be just disabled for all cases. See #L232